interface/reports/reports_controllers/ChartLocationActivityController.php

```diff
@@ -0,0 +1,116 @@
+<?php
```

Comment:
NOt real sure we even need this report anymore.  It is a good template for the new methodology though...